packages/frontend/src/main/components/common/NoDataPlaceholder.vue

```diff
@@ -190,13 +190,14 @@ export default {
         type: 'action'
       })
```

Comment:
This ` this.$mixpanel.track` can be removed, sorry I should have mentioned it, didn't think you were going to reuse this method ;)
Could actually opt for a `:href` on the button directly if you prefer ;)